config/elastic-apm-laravel.php

```diff
@@ -2,7 +2,7 @@
 
 return [
     // Sets whether the apm reporting should be active or not
-    'active' => env('APM_ACTIVE'),
+    'active' => env('ELASTIC_APM_ENABLED', env('APM_ACTIVE', true)),
```

Comment:
I have one concern about this implementation. In our documentation we state the following:

>The following options are still supported by this package and will take precedence over their ELASTIC_APM_* counterparts if present.

But with the actual change, it will actually so the opposite right? If both `ELASTIC_APM_*` and `APM_*` variables are present, `ELASTIC_APM_*` variables will take precedence, and might have unexpected consequences for some users.

Could this work instead, or not really?
```suggestion
    'active' => env('APM_ACTIVE', env('ELASTIC_APM_ENABLED', true)),
```

Reply:
You are correct, that would reverse the documented behavior. I don't see why flipping them would be an issue. Thanks for catching that.